fabric-resource-loader-v0/src/main/java/net/fabricmc/fabric/impl/resource/loader/ModResourcePackCreator.java

```diff
@@ -49,12 +51,14 @@ public void register(Consumer<ResourcePackProfile> consumer, ResourcePackProfile
 			}
 
 			ResourcePackProfile resourcePackProfile = ResourcePackProfile.of("fabric/" + ((ModResourcePack) pack).getFabricModMetadata().getId(),
-					false, () -> pack, factory, ResourcePackProfile.InsertionPosition.TOP,
+					true, () -> pack, factory, ResourcePackProfile.InsertionPosition.TOP,
```

Comment:
Is this boolean argument `alwaysEnabled` or `pinned`? I recall pack profiles have two boolean params and i forgot which one this is

Reply:
This boolean argument is `alwaysEnabled`, as it's the "mod" resource pack it should always be enabled and replace the old PR BoundaryBreaker did to do that.